amgut/db/patches/0040.sql

```diff
@@ -0,0 +1,220 @@
+-- Sep. 9, 2016
+-- Plate Mapper test 01
```

Comment:
Can you change the comment? This is not a test :-) Add a short description of what is being added and why, it is usually really useful to troubleshoot bugs in the future.